libevmasm/AssemblyItem.cpp

```diff
@@ -56,6 +57,51 @@ pair<size_t, size_t> AssemblyItem::splitForeignPushTag() const
 	return make_pair(subId, tag);
 }
 
+string AssemblyItem::toStringInHex(u256 _value)
```

Comment:
Don't we already have a utility function for that? In any case, please turn this into a function inside an anonnymous namespace at least.

Reply:
Yes, but I was not able to put it in exactly the needed hex format with the other utility functions. Moved to anonymous namespace.